.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+
+    tags:
+      - '*'
+
+  pull_request:
+
+env:
+  MAIN_REPO: Oceans-1876/challenger-portal
+
+jobs:
+
+# ----------------------------------------------------------------------
+  # NODE BUILD
+  # ----------------------------------------------------------------------
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
```

Comment:
We should set this to `true` so we don't continue with a broken build.

Reply:
If we don't intend to use the matrix, I'd agree. If we use the matrix of node JS, then it would be worthwhile to see where it fails and for which versions of node.